Sources/BrowserServicesKit/UserScript/UserScript.swift

```diff
@@ -73,11 +74,23 @@ extension UserScript {
     static func makeWKUserScript(source: String, injectionTime: WKUserScriptInjectionTime,
                                  forMainFrameOnly: Bool,
                                  requiresRunInPageContentWorld: Bool = false) -> WKUserScript {
+        let hash = SHA256.hash(data: Data(source.utf8)).hashValue
+
+        // This prevents the script being executed twice which appears to be a WKWebKit issue for about:blank frames when the location changes
+        let sourceOut = """
+        (() => {
+            if (window.navigator._duckduckgoloader_ && window.navigator._duckduckgoloader_.includes('\(hash)')) {return}
```

Comment:
can we make this check `Array.isArray()` instead - just to ensure `.includes()` is valid

Reply:
What's the rationale?

I'm only checking the presence as it's not defined first time round, the definition is always within the same file.

I don't think we should be defending against the page if that's your intent here; we'd need to take prototypes and prevent overriding etc.

Reply:
Resolved over MM :)